mod-finance/schemas/fy_finance_data.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Finance data for a fiscal year",
+  "type": "object",
+  "properties": {
+    "fiscalYearCode": {
+      "description": "The code of the fiscal year",
+      "type": "string"
+    },
+    "fundId": {
+      "description": "Fund UUID",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "fundCode": {
+      "description": "A unique code associated with the fund",
+      "$ref": "../../common/schemas/fund_code.json"
+    },
+    "fundName": {
+      "description": "The name of this fund",
+      "type": "string"
+    },
+    "fundStatus": {
+      "description": "The current status of this fund",
+      "type": "string",
+      "enum": [
+        "Active",
+        "Frozen",
+        "Inactive"
+      ]
+    },
+    "fundTags": {
+      "type": "object",
+      "description": "Arbitrary tags associated with this fund",
+      "$ref": "../../../raml-util/schemas/tags.schema"
+    },
```

Comment:
also lets add fundDescription per this screen
![image](https://github.com/user-attachments/assets/5a77468c-ec97-4ea5-b0ba-fb4b1e7740e3)